pgmpy/readwrite/ProbModelXML.py

```diff
@@ -570,7 +570,7 @@ def _add_element(potential, var, potential_tag):
             For example: {'role': 'Utility',
                           'Variables': ['D0', 'D1', 'C0', 'C1'],
                           'type': 'Tree/ADD',
-                          'UtilityVaribale': 'U1'}
+                          'UtilityVariable': 'U1'}
```

Comment:
@ankurankan Is this done on purpose? Or in code at line [947](https://github.com/pgmpy/pgmpy/pull/989/files#diff-10c1e53f55c8c830573e5a2c0014bce8L947) mistake has been repeated. I'm not able to understand.

Reply:
I purposefully did not change that field name to avoid changing the API. If you would like, I can change it there too

Reply:
@JustinTervala @khalibartan I think that is a typo and should be fixed.